ni_measurementlink_service/_internal/parameter/serialization_strategy.py

```diff
@@ -54,6 +56,53 @@ def vector_encoder(field_index):
     return vector_encoder
 
 
+local_int2byte = struct.Struct(">B").pack
+
+
+def _encode_varint(write, value, unused_deterministic=None):
+    bits = value & 0x7F
+    value >>= 7
+    while value:
+        write(local_int2byte(0x80 | bits))
+        bits = value & 0x7F
+        value >>= 7
+    return write(local_int2byte(bits))
```

Comment:
This is copied from google.protobuf. 

1. Do we really need to copy it?
2. If we really need to copy it, we should keep it in a separate module with a comment about its origin.
3. Can we copy it without restructuring it as much?

Reply:
Moved to _message.py with comments about where they came from.

Reply:
3. I meant copy `_VarintEncoder` so what's local stays local